.travis.yml

```diff
@@ -1,35 +1,36 @@
-language: cpp
+language: python
 
-os: 
-    - osx
-    - linux 
-
-env:
-    matrix:
-        - TEST_SUITE="bin/py.test -n 2 -vvs --ignore=tests/scancode --ignore=tests/extractcode --ignore=tests/licensedcode --ignore=tests/cluecode --ignore=tests/packagedcode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/scancode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/extractcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/licensedcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/cluecode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/packagedcode"
-        - TEST_SUITE="./etc/release/release.sh"
+python:
+  - 2.7.13
 
+matrix:
+  include:
+    - os: osx
+      language: generic
+      python:
+      osx_image: xcode7.3
 
 install:
-    - ./configure
+  - ./configure
+  - source bin/activate
+  - pip install pytest-cov coveralls
```

Comment:
Pushed an update, used `./bin` prefix everywhere. Added `./` because `./configure` and `./etc/release/release.sh` use it and I went consistent.